experiments/configs/hugging-face/eval_configs/eval_pipeline_config.yaml

```diff
@@ -0,0 +1,8 @@
+model: hf-causal
+model_args: "pretrained=/fsx/proj-chemnlp/experiments/checkpoints/finetuned/full_1b/checkpoint-final"
+# "pretrained=EleutherAI/pythia-1b"
```

Comment:
is `model_args` when you load locally and `pretrained` for loading from HF?

Reply:
No, it should be always be model_args and the a string 'pre-trained: [path]'. I have just made a change to make this more clear. Other args can be passed in as part of the string too because it gets split up and processed into separate model args, but I haven't looked into the others yet.